CHANGELOG.md

```diff
@@ -5,7 +5,8 @@
 ### Fixes
 
 - Relocate fee sanity check and make it relative to each transaction (rather than absolute) ([CO-446](https://iohk.myjetbrains.com/youtrack/issue/CO-446) [#3993](https://github.com/input-output-hk/cardano-sl/pull/3993)
-
+- Change how block retrieval worker's conversations are organised, so that the worker can recover from IOExceptions ([CDEC-658](https://iohk.myjetbrains.com/youtrack/issue/CDEC-658) [#3994](https://github.com/input-output-hk/cardano-sl/pull/3994))
```

Comment:
I'd reword this as

"Correction to block retrieval concurrency and exception handling, so that it will recover in case of certain network failure conditions."